packages/icons/src/normalize.svg

```diff
@@ -0,0 +1,40 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g fill="none" fill-rule="evenodd" transform="translate(2.56 -.656)">
+    <g fill-rule="nonzero" transform="translate(7 13)">
+      <polygon fill="#000" points=".357 3.655 3.688 3.655 3.688 .323 .357 .323"/>
```

Comment:
style properties like fill should be set up on a g element along with a css class wich allow us to target element.